components/scream/src/diagnostics/longwave_cloud_forcing.cpp

```diff
@@ -0,0 +1,75 @@
+#include "diagnostics/longwave_cloud_forcing.hpp"
+
+namespace scream
+{
+
+// =========================================================================================
+LongwaveCloudForcingDiagnostic::LongwaveCloudForcingDiagnostic (const ekat::Comm& comm, const ekat::ParameterList& params)
+  : AtmosphereDiagnostic(comm,params)
+{
+  // Nothing to do here
+}
+
+// =========================================================================================
+void LongwaveCloudForcingDiagnostic::set_grids(const std::shared_ptr<const GridsManager> grids_manager)
+{
+  using namespace ekat::units;
+  using namespace ShortFieldTagsNames;
+
+  auto Q = kg/kg;
```

Comment:
The units `Q` and `s2` are not actually used here, so we can remove these lines.